rtl/include/eclhelper.hpp

```diff
@@ -721,7 +721,7 @@ interface ICodeContext : public IResourceContext
 
     // Exception handling
 
-    virtual void addWuException(const char * text, unsigned code, unsigned severity, const char * source) = 0;
+    virtual void addWuException(const char * text, unsigned code, unsigned severity, unsigned audience, const char * source) = 0;
```

Comment:
This will invalidate existing generated code - so the compatible version on line 47/48 will need to be increased.